Mathlib/Tactic/TypeStar.lean

```diff
@@ -1,7 +1,7 @@
 /-
-Copyright (c) 2023 Matthew Ballard. All rights reserved.
+Copyright (c) 2023 Matthew Robert Ballard. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Matthew Ballard
+Authors: Matthew Robert Ballard
```

Comment:
```suggestion
Authors: Kyle Miller
```
I am not sure how I (alone!) ended up here but it was probably due to my general confusion.